### PR TITLE
Remove VT Courts Online code

### PR DESCRIPTION
**Pull request by jakedurell**
Removed the code and sample files for the now-defunct VT Courts Online site. Added a new sample file modified from an Odyssey docket and updated the root README (which drives the github pages project site).

### CODE REVIEW

File: extensionDirectory/popup.js [hunk 1]
```diff
@@ -7,20 +7,15 @@ function initListeners() {
...
-        parsedData = getVTCOPetitionerInfo(rawDocketData);
-        break;
-      }
       // ODYSSEY
       case 'publicportal.courts.vt.gov': {
         parsedData = getOdysseyPetitionerInfo(rawDocketData);
         break;
       }
-      // DEMO & DEV: currently we only have demo samples for VCOL dockets
+      // DEMO & DEV: TODO - currently we only have demo samples for VCOL dockets
```
Comment (doub1ejack) [inline extensionDirectory/popup.js:15]: It looks like the VTCO sample files haven't been changed or removed. We should probably do that as part of this PR too.

**Review by doub1ejack — CHANGES_REQUESTED** [review]
Before we remove the code, it would be nice to put together some good Odyssey sample dockets and make sure the [Demo section of the README](https://codeforbtv.org/expunge-vt/#:~:text=although%20you%20can%E2%80%99t%20generate%20real%20petitions%20without%20data%20access%2C%20you%20can%20give%20it%20a%20test%20run%20on%20these%20three%20sample%20dockets.) still works smoothly. (I often point people to that page when they're trying to understand exactly how the extension works.)

**Comment by doub1ejack** [comment]
Note, I cherry-picked the two text related commits (`typo interests of justice fix` and `amend sealing petition`) to another branch, merged that into master so they could be deployed separately, and merged master back into this branch. 

Now this branch only has the VT Courts Online removal changes. I'll publish the typo/text fixes from master to production now...

**Comment by jakedurell** [comment]
@doub1ejack - nice work this morning. Do we want to put that warning in? Are you still getting that deploy error?

**Pull request merged**